Lowercase the Block Editor and Classic Editor strings

garrett-eclipse opened the pull request:
To match the core convention for lowercasing the block editor and classic editor strings unless it's a title or referring to Classic Editor plugin
Reference - https://make.wordpress.org/core/handbook/best-practices/spelling/

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

kristastevens commented:
@yaelirub, thanks for the ping! Is there specific text you'd like us to look at? If this is more a philosophical question about approach, then yes, we'd agree that consistency with the Make WordPress Core Handbook is the way to go.

yaelirub commented:
Thanks for looking into this, @kristastevens .
Yes, it's more of a philosophical question :)

yaelirub commented on WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift:
```diff
@@ -15,10 +15,10 @@ extension GutenbergViewController {
+            comment: "Popup content about why this post is being opened in block editor"
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in block editor")
```
Should this be "Block editor enabled"?

kristastevens replied:
I think so, yes. We use sentence case as a default.

garrett-eclipse replied:
Thanks @yaelirub & @kristastevens updating coming in

garrett-eclipse commented:
Thanks @yaelirub I've updated that one string

garrett-eclipse commented:
Latest commit to branch is to resync with upstream/develop

jkmassel commented on WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift:
```diff
@@ -15,7 +15,7 @@ extension GutenbergViewController {
         /*if mode == .richText {
             // NB : This is a candidate for plurality via .stringsdict, but is limited by https://github.com/wordpress-mobile/WordPress-iOS/issues/6327
             let textCounterTitle = String(format: NSLocalizedString("%li words, %li characters", comment: "Displays the number of words and characters in text"), richTextView.wordCount, richTextView.characterCount)
-            
```
Can we remove this whitespace change?

garrett-eclipse replied:
I can't seem to whenever I introduce and save it auto-strips. Should it even be there though as no other empty lines have white space at the beginning like that

jkmassel replied:
Hmmm, good point. It's not ideal to have this in an unrelated PR, but it's even less ideal to have it in the code in the first place. I'm ok to remove it!

garrett-eclipse commented:
> Can we please remove changes to `WordPress/Resources/AppStoreStrings.po` before merging?

Sorry about that tried to sync with develop and resolve conflicts didn't go well so I rebased and removed that commit.

jkmassel commented:
@etoledom – I merged this branch with `develop` – as of https://github.com/wordpress-mobile/WordPress-iOS/pull/11921/commits/1c30b4e0439f8748ed06fe17d8ec4be70b192c23, this branch should align with https://github.com/wordpress-mobile/WordPress-iOS/blob/dbe265afa5528c918270785326737805499a825c/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController%2BInformativeDialog.swift#L8-L11

Looking for your 👍 on that prior to merge, if it's not too much trouble? :)

jkmassel commented:
I fixed up a merge issue, but unfortunately, Hound is unhappy now, which prevents us from merging. To resolve the line length issue, I'd suggest using the syntax from https://github.com/wordpress-mobile/WordPress-iOS/blob/dbe265afa5528c918270785326737805499a825c/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController%2BInformativeDialog.swift#L8-L11

Once that's fixed, and we get the 👍 from @etoledom, I'm good to :shipit:

garrett-eclipse commented:
Hi @jkmassel is that 'To resolve the line length issue, I'd suggest using the syntax from' item addressed to me or @etoledom let me know if anything is required from me and I can make an update

jkmassel commented:
Ah sorry for the confusion! If you're able to implement the new syntax, that would be fantastic (and Hound will really appreciate it too!)

garrett-eclipse commented:
k @jkmassel I'll take a look over the weekend here, gotta run and get the kids

garrett-eclipse commented on WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift:
```diff
@@ -9,7 +9,7 @@ extension GutenbergViewController {
             comment: "Popup content about why this post is being opened in block editor"
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString("Block editor enabled", comment: "Popup title about why this post is being opened in block editor")
```
addressed in 6409735

garrett-eclipse commented on WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift:
```diff
@@ -3228,7 +3228,7 @@ extension AztecPostViewController {
 
     struct MoreSheetAlert {
-        static let gutenbergTitle = NSLocalizedString("Switch to Block Editor", comment: "Switches from the Classic Editor to Block Editor.")
+        static let gutenbergTitle = NSLocalizedString("Switch to block editor", comment: "Switches from the classic editor to block editor.")
```
addressed in 6409735

garrett-eclipse commented on WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift:
```diff
@@ -94,7 +94,10 @@ extension GutenbergViewController {
     private struct MoreSheetAlert {
-        static let classicTitle = NSLocalizedString("Switch to Classic Editor", comment: "Switches from Gutenberg mobile to the Classic editor")
+        static let classicTitle = NSLocalizedString(
+            "Switch to classic editor", 
```
addressed in 4c8a136

garrett-eclipse commented on WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift:
```diff
@@ -9,7 +9,10 @@ extension GutenbergViewController {
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString(
+            "Block editor enabled", 
```
addressed in 4c8a136

garrett-eclipse commented on WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift:
```diff
@@ -3228,7 +3228,10 @@ extension AztecPostViewController {
     struct MoreSheetAlert {
-        static let gutenbergTitle = NSLocalizedString("Switch to Block Editor", comment: "Switches from the Classic Editor to Block Editor.")
+        static let gutenbergTitle = NSLocalizedString(
+            "Switch to block editor", 
```
addressed in 4c8a136

garrett-eclipse commented:
Alright, I think I appeased the beast ;)
"No violations found. Woof!"

Can you take a look @jkmassel 
As to your unresolved change request I was addressed in when I rebased. But because the file isn't part of the PR anymore so the change request conversion is gone and I can't mark resolved. Referring to this;
> Can we please remove changes to WordPress/Resources/AppStoreStrings.po before merging?
* did this by rolling back the commit.

jkmassel commented:
This is ready to merge. Thanks @garrett-eclipse!!

garrett-eclipse commented:
Awesome thanks for getting it to the endzone @jkmassel
